packages/adapter-cloudflare/index.js

```diff
@@ -24,6 +24,12 @@ export default function (options = {}) {
 	return {
 		name: '@sveltejs/adapter-cloudflare',
 		async adapt(builder) {
+			if (!options.routes && existsSync('_routes.json')) {
```

Comment:
should this validation occur whenever `_routes.json` is present independent of `!options.routes`?

Reply:
Yes, I believe this should be the best approach.

I wanted to avoid to throwing an error for who had both of them configured (and so a working environment) to suddenly throwing an error, but I didn't know which road I had to take, so I went for the "safest" one :)

I'll remove the check on `!options.routes` soon, thanks again for the review!